bin/validate

```diff
@@ -36,7 +36,7 @@ foreach ($finder as $file) {
     }
 
     $validator->reset();
-    $validator->check($yaml->schema, $jsonSchema);
+    $validator->check($yaml, $jsonSchema);
```

Comment:
Not sure this & line 60 is right. Script exits with 0, but with the message `Json Pointer '/schema' references a nonexistent value`.

Reply:
https://github.com/elifesciences/pattern-library/blob/3f04e2b1f0c5e2a3d267368c48866765b6e83945/source/_patterns/00-atoms/components/author-details.yaml#L38 (and similar) need to lose their pointers (=fragments), since it doesn't exist anymore.

Reply:
Running validate now on: https://github.com/elifesciences/pattern-library/pull/892/files#diff-460b5554de13b09ccc95e53925083a87R38 now gives `Pointer is not a string` (exit code 0).

Reply:
Fixed in 77d75aa3835769bb23698f523914a303aab1596f.

Reply:
(Though need to look at the exit code.)